internal/outpost/radius/handle_access_request.go

```diff
@@ -24,6 +24,10 @@ func (rs *RadiusServer) Handle_AccessRequest(w radius.ResponseWriter, r *RadiusR
 	fe.Answers[flow.StagePassword] = rfc2865.UserPassword_GetString(r.Packet)
 	if r.pi.MFASupport {
 		fe.CheckPasswordInlineMFA()
+	} else {
+		// If code-based MFA is disabled StageAuthenticatorValidate answer is set to StagePassword answer.
+		// This allows flows with only a mfa stage
+		fe.Answers[flow.StageAuthenticatorValidate] = fe.Answers[flow.StagePassword]
```

Comment:
I think this should either be changed for LDAP and Radius, and/or maybe it should be an option to configure...although I don't think this would be a breaking change to the current behaviour

Reply:
No this isn't a breaking change at all but it improves usability of the radius outpost.

I see three options to get this done better:
1) Implement this as an option (as you suggested) for both `LDAP` and `Radius` providers. 
But I think this would confuse users because of the already implemented code-based MFA option
2) Analyze the connected flow and fill answers accordingly. I'm not quite sure if this can be done at all and if it would cover all possible flows
3) Refactor [CheckPasswordInlineMFA](https://github.com/goauthentik/authentik/blob/0e1646ca1b651b7b0eeca02a62afa7620d9d690f/internal/outpost/flow/solvers_mfa.go#L15C33-L15C33) to something like `SetupFlowAnswers (string password, bool mfa)` and use it for both outposts. 

In my eyes we should go for option 3) but I leave the descision up to you @BeryJu